.travis.yml

```diff
@@ -17,6 +17,7 @@ go:
   - 1.7
 
 before_install:
+  - pip install --user awscli
```

Comment:
I added this because the CLI prints a different message if the AWS CLI hasn't been installed than the one it prints if the AWS CLI is installed but not configured. I don't know how to mock-toggle this in local tests, so in other words, the test output will be different locally (since we all have the AWS CLI installed) vs in Travis (which doesn't).